crates/sui-config/src/p2p.rs

```diff
@@ -40,7 +40,7 @@ pub struct P2pConfig {
 }
 
 fn default_listen_address() -> SocketAddr {
-    "0.0.0.0:8080".parse().unwrap()
+    "0.0.0.0:8084".parse().unwrap()
```

Comment:
This came up recently:
https://github.com/MystenLabs/sui/issues/20971
https://github.com/MystenLabs/sui/pull/20973

Reply:
Great context. Do you have opinions on how this should be updated?

Reply:
I can't think of any issues this would cause for an operator who isn't explicitly setting this value